ivc/src/plonkish_lang.rs

```diff
@@ -130,8 +129,8 @@ impl<G: KimchiCurve, const N_COL: usize, const N_CHALS: usize, const N_ALPHAS: u
     }
 }
 
-impl<G: KimchiCurve, const N_COL: usize, const N_CHALS: usize, const N_ALPHAS: usize> Instance<G>
-    for PlonkishInstance<G, N_COL, N_CHALS, N_ALPHAS>
+impl<G: CommitmentCurve, const N_COL: usize, const N_CHALS: usize, const N_ALPHAS: usize>
```

Comment:
Also changing this to `CommitmentCurve` as `bn254` does not implement the trait `KimchiCurve`. And nothing from the trait `KimchiCurve` is required.